src/Microsoft.TestPlatform.CrossPlatEngine/DataCollection/DataCollectionLauncher.cs

```diff
@@ -20,13 +19,8 @@ namespace Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.DataCollection
     internal class DataCollectionLauncher : IDataCollectionLauncher
```

Comment:
Please follow the convention:
* DefaultDataCollectionLauncher
* DotnetDataCollectionLauncher